sdk/spring/azure-spring-cloud-test-eventhubs/src/test/java/com/azure/spring/eventhubs/health/HealthBindersIT.java

```diff
@@ -0,0 +1,101 @@
+            app.property("spring.cloud.stream.bindings.supply-out-0.destination", "event-hub-name");
+            app.property("spring.cloud.stream.default.producer.errorChannelEnabled", "true");
+            app.property("spring.cloud.stream.eventhub.bindings.consume-in-0.consumer.checkpoint-mode", "MANUAL");
+            app.property("spring.cloud.stream.function.definition", "consume;supply");
+
+            app.property("management.endpoint.health.show-details", "always");
+            app.property("management.endpoints.web.exposure.include", "*");
+            app.property("management.health.binders.enabled", "true");
+
+            app.start();
+
+            final String response = REST_TEMPLATE.getForObject(
+                "http://localhost:" + app.port() + "/actuator/health/binders", String.class);
+            assertEquals("{\"status\":\"UP\",\"components\":{\"eventhub\":{\"status\":\"UP\"}}}", response);
+
+            LOGGER.info("response = {}", response);
+        }
+        LOGGER.info("testSpringBootActuatorHealth end.");
+    }
+
+
+    @SpringBootApplication
+    public static class Application {
+
+        private static final Logger LOGGER = LoggerFactory.getLogger(Application.class);
+
+        @Bean
+        public Consumer<Message<String>> consume() {
+            return message -> {
+                Checkpointer checkpointer = (Checkpointer) message.getHeaders().get(CHECKPOINTER);
+                LOGGER.info("New message received: '{}', partition key: {}, sequence number: {}, offset: {}, enqueued"
+                        + " time: {}",
+                    message.getPayload(),
+                    message.getHeaders().get(EventHubHeaders.PARTITION_KEY),
+                    message.getHeaders().get(EventHubHeaders.SEQUENCE_NUMBER),
+                    message.getHeaders().get(EventHubHeaders.OFFSET),
+                    message.getHeaders().get(EventHubHeaders.ENQUEUED_TIME)
+                );
+
+                checkpointer.success()
+                            .doOnSuccess(success -> LOGGER.info("Message '{}' successfully checkpointed",
+                                message.getPayload()))
+                            .doOnError(error -> LOGGER.error("Exception found", error))
+                            .subscribe();
+            };
+        }
+
+        // Replace destination with spring.cloud.stream.bindings.consume-in-0.destination
+        // Replace group with spring.cloud.stream.bindings.consume-in-0.group
+        @ServiceActivator(inputChannel = "{destination}.{group}.errors")
+        public void consumerError(Message<?> message) {
+            LOGGER.error("Handling customer ERROR: " + message);
+        }
+
+        // Replace destination with spring.cloud.stream.bindings.supply-out-0.destination
+        @ServiceActivator(inputChannel = "{destination}.errors")
+        public void producerError(Message<?> message) {
+            LOGGER.error("Handling Producer ERROR: " + message);
+        }
```

Comment:
We don't these for this test case, right?

Reply:
get it